gr-utils/python/modtool/modtool_newmod.py

```diff
@@ -90,6 +90,7 @@ def run(self):
                 s = open(f, 'r').read()
                 s = s.replace('howto', self._info['modname'])
                 s = s.replace('HOWTO', self._info['modname'].upper())
+                s = s.replace('This file is part of GNU Radio', 'This file was generated by gr_modtool, a tool from the GNU Radio framework \n * This file is a part of gr-'+self._info['modname'])
```

Comment:
Just change the `grlicense` template in templates.py

Reply:
I guess just changing the license in grlicense will not solve the issue. We will have to change all files in gr-newmod directory because the script recursively opens all files and rename howto and HOWTO to respective module name. Please correct me if I am wrong.

Reply:
You're right, just changing `grlicense` does not suffice. But if we want a different header, we should then change all the headers in gr-newmod -- after all, they too, are templates.

Reply:
...which you already did.